decidim-elections/app/commands/decidim/elections/admin/destroy_answer.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    module Admin
+      # This command is executed when the user destroys an Answer
+      # from the admin panel.
+      class DestroyAnswer < Rectify::Command
+        include ::Decidim::AttachmentMethods
+        include ::Decidim::GalleryMethods
```

Comment:
I'm just curious about what happens when you delete an answer that has images, images get also deleted? Where are they stored?

Reply:
All the attachments are in the `decidim_attachments` table, and that command behaviour is defined [here](https://github.com/decidim/decidim/blob/feat/elections-questions-answers/decidim-core/app/commands/decidim/gallery_methods.rb). Those images are automatically deleted when the resource is deleted, that is tested [here](https://github.com/decidim/decidim/blob/feat/elections-questions-answers/decidim-elections/spec/commands/decidim/elections/admin/destroy_answer_spec.rb#L42) and [here](https://github.com/decidim/decidim/blob/feat/elections-questions-answers/decidim-core/lib/decidim/core/test/shared_examples/admin_resource_gallery_examples.rb#L75)